mlir/lib/Dialect/Arith/Transforms/BufferViewFlowOpInterfaceImpl.cpp

```diff
@@ -0,0 +1,45 @@
+//===- BufferViewFlowOpInterfaceImpl.cpp - Buffer View Flow Analysis ------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/Arith/Transforms/BufferViewFlowOpInterfaceImpl.h"
+
+#include "mlir/Dialect/Arith/IR/Arith.h"
+#include "mlir/Dialect/Bufferization/IR/BufferViewFlowOpInterface.h"
+
+using namespace mlir;
+using namespace mlir::bufferization;
+using namespace mlir::arith;
```

Comment:
do you need all these usings? you seem to spell out arith for example anyway
(hard to see without compiler ;-)

Reply:
The `arith` one is not needed indeed.